acp/ideas_module.php

```diff
@@ -150,10 +207,20 @@ public function main($id, $mode)
 			}
 		}
 
-		if ($submit)
+		// Submit relevant log entries and output success message
+		if ($submit || $submit_forum_setup)
 		{
-			// Add option settings change action to the admin log
-			$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_SETTINGS_LOG');
+			if ($submit)
+			{
+				// Add option settings change action to the admin log
+				$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_SETTINGS_LOG');
+			}
+
+			if ($submit_forum_setup)
+			{
+				// Add forum setup action to the admin log
+				$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_FORUM_SETUP_LOG');
+			}
```

Comment:
I'm not sure that we need a separate log just for the utility, but if so, instead of 2 separate conditionals and duplicating the log, why not replace the above 2 conditions and the line below with something like:
```php
$message = ($submit_forum_setup) ? 'ACP_PHPBB_IDEAS_FORUM_SETUP_LOG' : 'ACP_PHPBB_IDEAS_SETTINGS_LOG';

$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, $message;
```